src/riscv/lib/src/pvm/tezos.rs

```diff
@@ -20,6 +23,14 @@ use tezos_smart_rollup_constants::riscv::SBI_TEZOS_INBOX_NEXT;
 use tezos_smart_rollup_constants::riscv::SBI_TEZOS_REVEAL;
 use tezos_smart_rollup_constants::riscv::SbiError;
 
+// temporary till added kernel_sdk constants
```

Comment:
Turn this into TODO comments for each of the constants.

Reply:
In fact, raise Linear issues for all and mention them in the `TODO` comment.